src/geometry/manifold/mfderiv.lean

```diff
@@ -242,6 +242,41 @@ def mdifferentiable (f : M → M') :=
 def local_homeomorph.mdifferentiable (f : local_homeomorph M M') :=
 (mdifferentiable_on I I' f f.source) ∧ (mdifferentiable_on I' I f.symm f.target)
 
+section open_emb
+
+/-- An open embedding is differentiable (for the charted-space structure induced by itself) -/
+lemma mdifferentiable_open_embedding {U : Type*} [topological_space U] [nonempty U]
```

Comment:
I think I'd put this a bit further down, just before `/-! ### Differentiable local homeomorphisms -/`